examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -170,29 +183,34 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 	 *
 	 * ```json
 	 *	{
-	 *		containerUrl: string,
+	 *		tenantId: string,
+	 *		documentId: string,
 	 *		externalTaskListId: string
 	 *	}
 	 * ```
-	 *
-	 * Note: Implementers choice --can choose to break up containerUrl into multiple pieces
-	 * containing tenantId, documentId and socketStreamURL separately and send them as a json
-	 * object. The URL also contains all this information so for simplicity I use the url here.
 	 */
 	expressApp.post("/register-session-url", (request, result) => {
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
Nit: Might be worth doing a single disable block for all of the lines doing request.body?.foo accesses.
E.g.
```typescript
		/* eslint-disable @typescript-eslint/no-unsafe-member-access */
		const documentId = request.body?.documentId as string;
		const tenantId = request.body?.tenantId as string;
		const externalTaskListId = request.body?.externalTaskListId as string;
		/* eslint-enable @typescript-eslint/no-unsafe-member-access */
```